pkgs/servers/sql/postgresql/ext/citus.nix

```
@@ -0,0 +1,36 @@
+{ lib, stdenv, fetchFromGitHub, postgresql, curl, lz4 }:
+
+stdenv.mkDerivation rec {
+  pname = "citus";
+  version = "11.3.0";
+
+  src = fetchFromGitHub {
+    owner  = "citusdata";
+    repo   = "citus";
+    rev    = "v${version}";
+    sha256 = "e+pkKFFFIATIoODRmOGFr0yFCYo2Q7mVJJ/zbtObJ30=";
```

The hash would need to be regenerated for this change I think

```suggestion
    hash = "sha256-e+pkKFFFIATIoODRmOGFr0yFCYo2Q7mVJJ/zbtObJ30=";
```